activiti-cloud-notifications-graphql-service/services/graphiql/src/main/resources/static/graphiql/graphiql.js

```diff
@@ -12282,7 +12282,7 @@ function extractLineClasses(type, output) {
     var prop = lineClass[1] ? "bgClass" : "textClass";
     if (output[prop] == null)
       { output[prop] = lineClass[2]; }
-    else if (!(new RegExp("(?:^|\s)" + lineClass[2] + "(?:$|\s)")).test(output[prop]))
```

Comment:
is this \s change intended ?

Reply:
yes, spotted by code ql https://github.com/Activiti/activiti-cloud/security/code-scanning/2